myosuite/envs/myo/myochallenge/chasetag_v0.py

```diff
@@ -429,6 +595,8 @@ def _setup(self,
                hills_range=(0,0),
                rough_range=(0,0),
                relief_range=(0,0),
+               velocity=1.0,
```

Comment:
change it to tuple to specify range 
Can we use sign of the velocity to specify chase / evade?

Reply:
changed to tuple, but didnt use sign to specify chase/evade, see above